### ISSUE TITLE
"Python has stopped working" pop up when using wait_cpu_usage_lower

### ISSUE THREAD

**Issue by furby-eb** [pywinauto/pywinauto#550]
Hi,
I'm facing the following problem now. When I'm trying to use **wait_cpu_usage_lower**, I'm getting the following pop up and my script is stopped:
"Python hash stopped working. A problem caused the program to stop working correctly. WIndows will close the program and notify you if a solution is available."

Please have a look, if I use it correctly.
```
    dlg = Application(backend="uia").start("C:\Program Files\App.exe")
    dlg.wait_cpu_usage_lower(threshold=2.5)
    dlg.window(title='Login')['Edit1'].Edit.set_edit_text('user')
```
I'm using backend="uia", pywinauto ver.0.6.5, python 3.7.0 (64-bit version), Windows 10(64-bit)
(yesterday I had 32-bit Python installed, today I came to conclusion that maybe this causes the problem, I reinstalled python to 64-bit version, but it still crashes on this. Do you have any idea what can be the problem? 

I'm running the app using normal user. I'm wondering if my user should have the special rights to be able to run wait_cpu_usage_lower function? What is so special about this function?

Thank you very much for your help!

**Comment by vasily-v-ryabov** [comment]
Hi @furby-eb I saw similar problems in unit tests of pywinauto specifically on Python 3.7.0 (both "bittedness"). Interestingly it was not crashed for Python 3.7 beta versions. Right now I can't exactly say it's Python bug or pywinauto issue hidden by previous Python implementations. I need some time to debug when I have a chance.

You may install Python 3.6.5 for now. Hope it's not critical for your project.

**Comment by vasily-v-ryabov** [comment]
Well, the root cause is that `pywin32==223` was built incorrectly. I can confirm the issue is reproduced on my PC with `pywin32==223` and it is gone after upgrading it by `C:\Python37_x64\Scripts\pip.exe install -U pywin32`.

Quote from pyWin32 release notes:

> * Built with a released version of Python 3.7, which fixes various date related
>   issues with Python 3.7

It explains the crash because it happens at line `win32process.GetProcessTimes(h_process)`. Diving into MSDN docs for [GetProcessTimes](https://docs.microsoft.com/en-us/windows/desktop/api/processthreadsapi/nf-processthreadsapi-getprocesstimes) we can see that many arguments have type `LPFILETIME` so this is date/time related definitely.

Pywinauto unit tests on Py3.7 were also magically fixed several days ago though AppVeyor is still using the same Python 3.7.0.

I think now we can close this issue.
